ghw/libghw.c

```diff
@@ -392,7 +457,10 @@ ghw_read_str (struct ghw_handler *h)
       h->str_table[i] = p;
       prev = h->str_table[i - 1];
       for (j = 0; j < prev_len; j++)
+      {
+	if (p >= p_end) goto ghw_read_str_err;
```

Comment:
Please use `err_ghw_read_str` for the label.

Reply:
Changed.